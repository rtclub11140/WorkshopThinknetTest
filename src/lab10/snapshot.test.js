import Adapter from 'enzyme-adapter-react-16';
import Enzyme, {shallow, mount, render} from 'enzyme';
import React from 'react'

import Button from './snapshot'

Enzyme.configure({adapter: new Adapter()});

it('converts basic button using shallow', () => {
  const shallowed = shallow(
    <Button className="class">
      <strong>Hello!</strong>
    </Button>,
  );

  expect(shallowed).toMatchSnapshot();
});

it('converts basic button using mount', () => {
  const mounted = mount(
    <Button className="class">
      <strong>Hello!</strong>
    </Button>,
  );

  expect(mounted).toMatchSnapshot();
});

it('converts basic button using render', () => {
  const rendered = render(
    <Button className="class">
      <strong>Hello!</strong>
    </Button>,
  );

  expect(rendered).toMatchSnapshot();
});