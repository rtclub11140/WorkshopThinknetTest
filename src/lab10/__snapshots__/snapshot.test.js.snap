// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`converts basic button using mount 1`] = `
<Button
  className="class"
>
  <button
    className="class"
  >
    <strong>
      Hello!
    </strong>
  </button>
</Button>
`;

exports[`converts basic button using render 1`] = `
<button
  class="class"
>
  <strong>
    Hello!
  </strong>
</button>
`;

exports[`converts basic button using shallow 1`] = `
<button
  className="class"
>
  <strong>
    Hello!
  </strong>
</button>
`;
